test/tests/functional/pbs_multi_sched.py

```diff
@@ -136,6 +136,17 @@ def set_scheduling(self, scheds=None, op=False):
                 self.server.manager(MGR_CMD_SET, SCHED, {'scheduling': op},
                                     id=each)
 
+    def delete_sched(self, sched_name):
+        """
+        Helper function to delete sched"
+        """
+        self.scheds[sched_name].terminate()
+        sched_log = self.scheds[sched_name].attributes['sched_log']
+        sched_priv = self.scheds[sched_name].attributes['sched_priv']
+        self.du.rm(path=sched_log, sudo=True, recursive=True, force=True)
+        self.du.rm(path=sched_priv, sudo=True, recursive=True, force=True)
+        self.server.manager(MGR_CMD_DELETE, SCHED, id=sched_name)
```

Comment:
This helper function is fine where it is, but it could be useful in the PTL scheduler class.  We already have a create_scheduler() function which makes them.  Having a function which deletes one would be useful too.

Reply:
Hi @bhroam , we have already function which deletes multisched in ptl_server.py file "delete_sched_config" but this deletes all the schedulers in setup. I had to write to delete individual sched.

Reply:
Maybe break the function into 2 parts?  One to delete one sched, and the other to loop over all the scheds and call the other function?  It'll be more useful in the future, but isn't necessary now.  Do you want to do that, or should I merge?

Reply:
Hey Bhroam, I have created a separate ticket for these changes.